exonum/src/helpers/fabric/details.rs

```diff
@@ -459,8 +528,26 @@ impl Command for GenerateNodeConfig {
         let services_public_configs = new_context.get(keys::SERVICES_PUBLIC_CONFIGS).unwrap();
         let services_secret_configs = new_context.get(keys::SERVICES_SECRET_CONFIGS);
 
-        let (consensus_public_key, consensus_secret_key) = crypto::gen_keypair();
-        let (service_public_key, service_secret_key) = crypto::gen_keypair();
+        let consensus_public_key = {
+            let passphrase = Self::get_passphrase(&new_context, SecretKeyType::Consensus);
+            create_secret_key_file(&consensus_secret_key_path, &passphrase.0)
+        };
+        let service_public_key = {
+            let passphrase = Self::get_passphrase(&new_context, SecretKeyType::Service);
+            create_secret_key_file(&service_secret_key_path, &passphrase.0)
+        };
+
+        let pub_config_folder = Path::new(&pub_config_path).parent().unwrap();
```

Comment:
`pub_config_path` is path to configuration file how can it be "/"?
I put `expect` instead of `unwrap` here, also I renamed `folder` to `dir` in the name of the variable)